rpcs3/rpcs3qt/debugger_frame.cpp

```diff
@@ -1061,13 +1061,25 @@ void debugger_frame::ShowGotoAddressDialog()
 
 	if (dlg->exec() == QDialog::Accepted)
 	{
-		const u32 address = EvaluateExpression(expression_input->text());
-		m_debugger_list->ShowAddress(address, true);
+		PerformGoToRequest(expression_input->text());
 	}
 
 	dlg->deleteLater();
 }
 
+bool debugger_frame::PerformGoToRequest(const QString& text_argument)
```

Comment:
It makes no sense to return a bool if it's never used

Reply:
Fixed.